docs/4.2/architecture/teleport_architecture_overview.md

```diff
@@ -92,7 +92,7 @@ steps are explained below the diagram.
     clarity, but Teleport services do not have to run on separate nodes.
     Teleport can be run as a binary on a single-node cluster with no external
     storage backend. We demonstrate this minimal setup in the [Quickstart
-    Guide](../guides/quickstart).
```

Comment:
We validate the docs before each push with `milv` (markdown internal link validator) so I'm unsure how a broken link would get in, but good catch!